### DISCUSSION TITLE
服务返回错误，说明请求过于频繁或超出额度限制，请稍后再试。

### DISCUSSION THREAD

**Discussion by wthinkit** [BurnHair/docs#105] [分享讨论]
刚买了点额度，沉浸翻译并发配置改成30，翻译页面报请求频繁，是要等一会才会生效为付费的并发吗？
![image](https://github.com/BurnHair/docs/assets/16713368/efd37605-2129-47fd-b950-3cc4fa30b405)

![image](https://github.com/BurnHair/docs/assets/16713368/8c4f05ea-cb80-4299-8b64-235eb1b4bcd2)

**Comment by BennyThink** [comment]
你已经是了，并发限制是50，再试试看呢？

**Reply by wthinkit** [reply]
嗯 可以啦
